bunsen-core-stu3/src/main/java/com/cerner/bunsen/definitions/stu3/Stu3StructureDefinitions.java

```diff
@@ -266,9 +266,16 @@ private boolean shouldTerminateRecursive(DefinitionVisitor visitor,
     } else if (element.getType().size() == 1
         && PRIMITIVE_TYPES.contains(element.getTypeFirstRep().getCode())) {
 
+      T primitiveConverter = visitor
```

Comment:
nit: All of the above `if` conditions can be on its own, no need for `else` block, as each of these `return` from the function.

Reply:
@karthisuku I agree we could move them to own if conditions, but to minimize the changes at this moment, I would like to leave them as is and refactor them in future PRs.